components/StyledTag.js

```diff
@@ -59,7 +59,12 @@ const StyledTagBase = styled.div`
   text-align: center;
   white-space: nowrap;
   letter-spacing: 0.06em;
-
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 80%;
+  &:hover {
+    max-width: 500px;
+  }
```

Comment:
This component is used in many places. Do we really want this everywhere? Did you check other places to make sure it did not create any regression?